src/components/productDetails/list.jsx

```diff
@@ -112,61 +113,126 @@ const List = ({
       }
     }
   }
+
+  var wrapperStyle = {
+    width: "100%",
+    overflow: "hidden"
+    // display: "flex",
+    // "flex-wrap": "nowrap",
+    // "overflow-x": "auto"
+  };
+  var innerStyle = {
+    width: "100%"
+  };
+
   return (
-    <div class="columns">
-      <div className="column is-1 is-vertical-centered">
-        {shownIndices[0] !== 0 ? (
-          <DirectionalButton
-            arrowDirection={"left"}
-            icon={"fas fa-chevron-left"}
-            onArrowClick={onArrowClick}
-          />
-        ) : null}
-      </div>
-      {listName === "Outfit" ? (
-        <AddToOutfitCard
-          pageProduct={pageProduct}
-          onClickButton={onClickButton}
-        />
-      ) : null}
-      {filterForShownItems(products, shownIndices).map((product, idx) => {
-        return (
-          <div className="column is-one-fifth">
-            <Card
-              key={idx}
-              listName={listName}
-              pageProduct={pageProduct}
-              product={product}
-              productImage={
-                filterForShownItems(productsImages, shownIndices)[idx] !==
-                undefined
-                  ? filterForShownItems(productsImages, shownIndices)[idx]
-                      .results[0].photos[0].thumbnail_url
-                  : null
-              }
-              onClickDetails={onClickDetails}
-              onClickButton={onClickButton}
-              productReviews={
-                filterForShownItems(productsReviews, shownIndices)[idx]
-              }
-            />
+    <>
+      <h1 className="title">{listName} Items</h1>
+      <div className="columns">
+        <div className="column is-narrow is-vertical-centered">
+          <span>
+            {shownIndices[0] !== 0 ? (
+              <DirectionalButton
+                arrowDirection={"left"}
+                icon={"fas fa-chevron-left"}
```

Comment:
after seeing all the references to "fas" in your code i finally looked up what that was. i had no idea that bulma has all of these icons preloaded!

Reply:
It doesn't actually. If you go look at our index.html file we are also loading font awesome.